aws_backendless_chef_ha.yml

```diff
@@ -0,0 +1,1052 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: >
+  AWS HA Chef Server 2.0 - Backendless Chef - Ubuntu Xenial Based
+  "Backendless" Chef utilizing AWS as Backend for PostgreSQL and Elasticsearch
+  Sets up the following:
+    Backendless Items
+    * Elasticsearch 2.3 Cluster
+    * PostgreSQL 9.5.4
+
+Parameters:
+
+  ## Domain Configuration
+  HostedZone:
+    Type: String
+    Default: domain.com
+    Description: must match a route53 hosted domain/zone
+
+  SSLCertificateARN:
+    Type: String
+    Default: 'arn:aws:iam::'
```

Comment:
I explain this a bit in the readme: https://github.com/HearstAT/cfn_backendless_chef#parameters